privval/file_deprecated.go

```diff
@@ -58,7 +58,7 @@ func (oldFilePV *OldFilePV) Upgrade(keyFilePath, stateFilePath string) *FilePV {
 
 	pvState := FilePVLastSignState{
 		Height:    oldFilePV.LastHeight,
-		Round:     oldFilePV.LastRound,
+		Round:     int32(oldFilePV.LastRound),
```

Comment:
What was this before? Do we know this is safe?

Reply:
this was an `int` before. I didn't want to change the `oldFilePV` to abide by the new type.